WebContent/WEB-INF/jsp/controlPanel.jsp

```diff
@@ -146,6 +146,9 @@
 									   <span class="submenuText"><input type="radio" name="size" value="v" > MP3/MP4 </span><br />
 									</fieldset>
 						</div>
+						<div title="Choose the frame no. for video thumbnail creation. Default is 100." class="specialmenuText">
```

Comment:
was this option to choose a video frame asked for? i'm not seeing it in the ticket description. apologies if I missed it.